src/transformers/modeling_utils.py

```diff
@@ -223,6 +224,15 @@ def set_quantized_state():
     finally:
         _is_quantized = False
 
+@contextmanager
+def set_zero3_state():
```

Comment:
can we have a small comment documentation about that (explaning the issue we fixed with this basically) 😉